tests/test_config.py

```diff
@@ -20,17 +19,24 @@ def _parse_config(self, config_str):
         return conf.export('dict')
 
     def test_load(self):
-        config_yaml = """
```

Comment:
@Yajo could you preserve the existing test untouched so we can have confidence the change preserves backward compatibility